### DISCUSSION TITLE
Any way to reduce bundle size?

### DISCUSSION THREAD

**Discussion by ha1fstack** [naver/egjs-flicking#769] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/61955474/209270605-412c0e95-dcdb-4bda-806d-ed5a4ac5abe9.png)
![image](https://user-images.githubusercontent.com/61955474/209270634-38de7995-e4a5-4087-b349-96d227dad63c.png)

Using flicking in NextJS environment, and I'm seeing like ~40kb gzipped for the whole bundle. Any way to optimize or tree shake it?

**Comment by malangfox** [comment, marked as answer]
Hello @plrs9816.

Our package does not provide a specific way to reduce the bundle size.
We're sorry for not being able to support any way to optimize the size at this moment.
